packages/vc-verification/src/resolver/Ipfs-credential-resolver.ts

```diff
@@ -81,6 +90,13 @@ export class IpfsCredentialResolver implements CredentialResolver {
    */
   async getVerifiableCredential(did: string, namespace: string) {
     const credentials = await this.credentialsOf(did);
+    credentials.map((credential) =>
```

Comment:
I think `forEach` is a bit more appropriate here we are not interested in the resulting array from the `map`
```suggestion
    credentials.forEach((credential) =>
```